### PR TITLE
Various updates to Readme - on nmoreau's readme-update-dev 

### PR DESCRIPTION
**Pull request by kailinr**
## 📝 Summary

- security audit more explicit
- added community relay list and relays documentation URLs
- mev-boost to MEV-Boost
- removed The Plan from readme <- curious if you agree, @metachris 
- added my name as contributor
- other small typos and language updates

## ✅ I have run these commands

* [x] `make lint`
* [x] `make test-race`
* [x] `go mod tidy`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,42 +13,54 @@ With MEV-Boost, validators can access blocks from a marketplace of builders. Bui
 
 ## How does MEV-Boost work?
 
-
-PoS node operators must run three pieces of software: a validator client, consensus client, and an execution client. MEV-boost is a sidecar for the Consensus Client, a separate piece of open source software, which queries and outsources block-building to a network of builders. Block builders prepare full blocks, optimizing for MEV extraction and fair distribution of rewards. They then submit their blocks to relays.
+PoS node operators must run three pieces of software: a Validator client, a Consensus client, and an Execution client. MEV-boost is a sidecar for the Consensus client, a separate piece of open source software, which queries and outsources block-building to a network of builders. Block builders prepare full blocks, optimizing for MEV extraction and fair distribution of rewards. They then submit their blocks to relays.
```
[the remark this answers is not in the document]
Comment (metachris) [reply README.md:16]: they should not be.

File: README.md [hunk 2]
```diff
@@ -80,39 +92,44 @@ Requires [Go 1.18+](https://go.dev/doc/install).
...
+
+Ensure you are downloading the most updated MEV-Boost release. Releases are available at https://github.com/flashbots/mev-boost/releases
 
 clone the repository and build it:
 
 ```bash
 git clone https://github.com/flashbots/mev-boost.git
 cd mev-boost
+# Checkout the most updated version based on tags. The example below is for MEV-Boost v1.3.1
+git checkout tags/v1.3.1
```
[the remark this answers is not in the document]
Comment (metachris) [reply README.md:112]: yes 👍

File: README.md (hunk 1 above)
Comment (metachris) [inline README.md:16]:
```suggestion
PoS node operators must run three pieces of software: a validator client, a beacon node, and an execution client. MEV-boost is a sidecar for the beacon node - a separate piece of open source software, which queries and outsources block-building to a network of builders. Block builders prepare full blocks, optimizing for MEV extraction and fair distribution of rewards. They then submit their blocks to relays.
```

File: README.md [hunk 3]
```diff
@@ -13,42 +13,54 @@ With MEV-Boost, validators can access blocks from a marketplace of builders. Bui
...
-* [mev-boost relay source code](https://github.com/flashbots/mev-boost-relay)
+* [MEV-Boost relay source code](https://github.com/flashbots/mev-boost-relay)
 * Specs:
   * [Builder API](https://ethereum.github.io/builder-specs)
   * [Flashbots Relay API](https://flashbots.notion.site/Relay-API-Spec-5fb0819366954962bc02e81cb33840f5)
 
 
 # Table of Contents
 
+- [Table of Contents](#table-of-contents)
```
[the remark this answers is not in the document]
Comment (metachris) [reply README.md:44]: certainly remove table of contents from the table of contents :)

File: README.md [hunk 4]
```diff
@@ -13,42 +13,54 @@ With MEV-Boost, validators can access blocks from a marketplace of builders. Bui
...
 - [Background](#background)
 - [Installing](#installing)
+  - [Binaries](#binaries)
+  - [From source](#from-source)
+    - [`go install`](#go-install)
```
Comment (metachris) [inline README.md:49]: remove 3rd level ToC

File: README.md (hunk 2 above)
Comment (metachris) [inline README.md:112]:
```suggestion
# Checkout a specific release (the default main branch includes not yet released code)
git checkout tags/TAG  # replace TAG with a release version of your choosing
```

File: README.md [hunk 5]
```diff
@@ -162,53 +179,40 @@ make build-portable
...
 
-Run mev-boost pointed at our [Sepolia Relay](https://builder-relay-sepolia.flashbots.net/):
+Run MEV-Boost pointed at our [Sepolia Relay](https://builder-relay-sepolia.flashbots.net/):
 
 ```
  ./mev-boost -sepolia -relay-check -relays https://0x845bd072b7cd566f02faeb0a4033ce9399e42839ced64e8b2adcfc859ed1e8e1a5a293336a49feac6d9a5edb779be53a@builder-relay-sepolia.flashbots.net
 ```
 
-
 #### `test-cli`
```
Comment (metachris) [inline README.md:213]:
```suggestion
### `test-cli`
```

File: README.md [hunk 6]
```diff
@@ -241,17 +245,12 @@ sequenceDiagram
     mev_boost-->>consensus: submitBlindedBlock response
 ```
 
-# The Plan
```
[the remark this answers is not in the document]
Comment (metachris) [reply README.md:244]: it's okay to remove from here, as this was outlining the plan up until the merge.

**Comment by metachris** [comment]
very nice. left a few suggestions and comments 👍

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kailinr) [reply README.md:44]: it is auto generated.. will have to remove header

[the remark this answers is not in the document]
Comment (kailinr) [reply README.md:44]: It is auto generated, I cannot remove it without removing header. Will remove header for now, feel free to commit it back in.

File: README.md (hunk 4 above)
Comment (kailinr) [reply README.md]: I cannot unless we reformat the ### markdown in the development sections, so I removed ### markdown.

File: README.md (hunk 2 above)
Comment (kailinr) [reply README.md]: removed already

**Comment by kailinr** [comment]
All changes corrected, ready for review!

File: README.md [hunk 7]
```diff
@@ -78,41 +84,44 @@ https://github.com/flashbots/mev-boost/releases
 
 Requires [Go 1.18+](https://go.dev/doc/install).
 
-### `go install`
+**`go install`**
```
Comment (metachris) [inline README.md:87]: keep previous header

File: README.md [hunk 8]
```diff
@@ -13,42 +13,48 @@ With MEV-Boost, validators can access blocks from a marketplace of builders. Bui
...
+  - [From source](#from-source)
+  - [From Docker image](#from-docker-image)
+  - [Systemd configuration](#systemd-configuration)
+  - [Troubleshooting](#troubleshooting)
 - [Usage](#usage)
-- [The Plan](#the-plan)
+    - [Mainnet](#mainnet)
+    - [Goerli testnet](#goerli-testnet)
+    - [Sepolia testnet](#sepolia-testnet)
+      - [`test-cli`](#test-cli)
```
Comment (metachris) [inline README.md:52]: test-cli should be h2, not h3 nested under spolia

File: README.md [hunk 9]
```diff
@@ -78,41 +84,44 @@ https://github.com/flashbots/mev-boost/releases
...
+**Clone and Build
+**
+Ensure you are downloading the most updated MEV-Boost release. Releases are available at https://github.com/flashbots/mev-boost/releases
 
 clone the repository and build it:
 
 ```bash
 git clone https://github.com/flashbots/mev-boost.git
 cd mev-boost
+# Build the checked-out version from https://github.com/flashbots/mev-boost/releases
```
Comment (metachris) [inline README.md:105]: that comment is not quite correct, please see my suggestion in the PR before and use that here

File: README.md [hunk 10]
```diff
@@ -78,41 +84,44 @@ https://github.com/flashbots/mev-boost/releases
...
 
-Install mev-boost with `go install`:
+Install MEV-Boost with `go install`:
 
 ```bash
 go install github.com/flashbots/mev-boost@latest
 mev-boost -help
 ```
 
-### Clone & build
```
Comment (metachris) [inline README.md:90]: keep the header

File: README.md [hunk 11]
```diff
@@ -13,42 +13,48 @@ With MEV-Boost, validators can access blocks from a marketplace of builders. Bui
...
-- [The Plan](#the-plan)
+    - [Mainnet](#mainnet)
+    - [Goerli testnet](#goerli-testnet)
+    - [Sepolia testnet](#sepolia-testnet)
+      - [`test-cli`](#test-cli)
 - [API](#api)
 - [Maintainers](#maintainers)
 - [Contributing](#contributing)
 - [Security](#security)
+  - [Security Audits](#security-audits)
```
Comment (metachris) [inline README.md:57]: why nested? may be outdated toc?

Comment (kailinr) [reply README.md:58]: toc is automatic, follows title markdown (## vs. ###)

Comment (metachris) [reply README.md]: Automatic by whom? You can still remove/cleanup before committing?

**Review by metachris — APPROVED** [review]

**Comment by metachris** [comment]
Thanks!

**Pull request merged**